code/__HELPERS/string_lists.dm

```diff
@@ -14,12 +14,12 @@ GLOBAL_LIST_EMPTY(string_lists)
 	return GLOB.string_lists[string_id] = values
 
 ///A wrapper for baseturf string lists, for support for non list values, and a stack_trace if we have major issues
-/proc/baseturfs_string_list(list/values, turf/baseturf_holder)
+/proc/baseturfs_string_list(list/values, turf/baseturf)
```

Comment:
A `baseturf` is a turf under another, or a list of.
This is not one, this is a turf which points to a baseturf, thus the suggestion of `baseturf_holder`.
The point here is to make variables clear and informative, can you fix that?

Reply:
Actually it's only 20 columns, I can be ok with this.